style.css

```diff
@@ -428,6 +433,80 @@ textarea.form-input::placeholder {
   width: 13.4rem;
 }
 
+.open-modal {
+  width: 91%;
+  background-color: #fff;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  padding: 1.6rem;
+  padding-bottom: 2.4rem;
+  margin: auto;
+  z-index: 5;
+  box-shadow: 0 0 20px rgb(84, 84, 84);
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  padding: 1.2rem 0;
+}
+
+.modal-header h2 {
+  font-family: Roboto, sans-serif;
+  font-size: 3.2rem;
+  font-weight: 700;
+}
+
+.open-modal .tags-box {
+  margin-top: 4.8rem;
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  gap: 2.4rem;
+  margin-top: 2.4rem;
+}
```

Comment:
- [OPTIONAL] It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 

- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  Kindly decrease the space between the h2 title and the ul tag box and add more space bettween the ul tag box and the popup image. 🎨 👍🏻 

- Yours

![image](https://github.com/Gh-Reza/Portfolio-mobile-version/assets/79058364/fb1e4cbf-6433-420b-aca2-177c84c2860f)


- Figma:

![image](https://github.com/Gh-Reza/Portfolio-mobile-version/assets/79058364/33806782-a2f6-461a-afd2-e6ee24cf90ef)